### DISCUSSION TITLE
Howto move master container

### DISCUSSION THREAD

**Discussion by MarcS1975** [nextcloud/all-in-one#5776] [Questions] [answered]
Hi - my AIO installation via docker compose works well but my docker folder is running out of space. Can I somehow move the AIO master container (residing incie /var/lib/docker) to a new location? I **do not** want to move my entire docker folder, just the AIO masetr container residing inside /var/lib/docker. I have already moved the AIO data folders but the AIO compose file says **that I cannot move the location of the master container.** Has any one found a solution? 

```
My Compose:

volumes:
      - nextcloud_aio_mastercontainer:/mnt/docker-aio-config # This line is not allowed to be changed as otherwise the built-in backup solution will not work
```

**Comment by szaimen** [comment, marked as answer]
Hi, lets continue in https://help.nextcloud.com/t/master-container-running-out-of-space/212890
